data/people/adam-nicholls.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "Adam Nicholls",
+    "description": "Goram &amp; Vincent",
```

Comment:
Totally OCD, but could we add a full stop after the description to make it a proper sentence? :D